Tests/SRGDataProviderNetworkTests/ContentServicesTestCase.m

```diff
@@ -118,6 +118,59 @@ - (void)testContentPageForTopic
     [self waitForExpectationsWithTimeout:30. handler:nil];
 }
 
+- (void)testContentPageForShow
```

Comment:
This method includes several cases. Can we break them down further?

Reply:
I like the idea. But currently, tests are on a scope case, which can group multi state cases (know and unknown ids, boolean `true` / `false` values, etc…).

If we are agree (iOS dev maintainers), we should definitely create an issue to do it on the whole tests in this class and probably the whole project.

Reply:
Issue created to push the idea on the whole library.
https://github.com/SRGSSR/srgdataprovider-apple/issues/54